### PR TITLE
Create ActionCaption component

### PR DESCRIPTION
**Pull request by ma12-co**
Upstreaming talk's `actioncaption`, as per the original @jenniferpiperek design

Screenshot from 2021-03-03 15-52-30

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/ActionCaption/ActionCaption.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+			type: String,
+			required: true,
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+	.app-navigation-caption {
+		color: var(--color-primary);
```
Comment (nickvergessen) [inline src/components/ActionCaption/ActionCaption.vue:54]: White on white incoming... Needs to use color-primary-element?

**Review by nickvergessen — COMMENTED** [review]
I think the color is too much inside an action component. Its okay in the navigation component but not in the action popover menu

**Comment by ma12-co** [comment]
> I think the color is too much inside an action component. Its okay in the navigation component but not in the action popover menu

Imo it creates a nice separation between actions and caption, but I don't have a strong opinion here. @jancborchardt  your call

**Review by skjnldsv — CHANGES_REQUESTED** [review]
Waiting fro @jancborchardt's return

**Comment by jancborchardt** [comment]
> > I think the color is too much inside an action component. Its okay in the navigation component but not in the action popover menu
> 
> Imo it creates a nice separation between actions and caption, but I don't have a strong opinion here. @jancborchardt your call

I would agree the color _and_ boldness is too much for just a caption – since the options themselves are the important thing. I would actually set the caption in `var(--color-text-maxcontrast)` (and possibly not bold) and use the color (and maybe the boldness, but probably not) for the selected option instead.

**Comment by ma12-co** [comment]
Allright, ready to review again

Screenshot from 2021-04-16 09-19-54

**Review by skjnldsv — APPROVED** [review]
Let's use variables :rocket:

**Review by skjnldsv — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
ohhh, a conflict!

**Pull request merged**